rules_gapic/csharp/csharp_gapic_pkg.bzl

```diff
@@ -74,11 +74,14 @@ _csharp_gapic_src_pkg = rule(
     implementation = _csharp_gapic_src_pkg_impl,
 )
 
-def csharp_gapic_assembly_pkg(name, deps, **kwargs):
+def csharp_gapic_assembly_pkg(name, deps, assembly_name = None, **kwargs):
+    package_dir = name
+    if assembly_name:
+        package_dir = "google-cloud-%s-%s" % (assembly_name, name)
```

Comment:
Is it possible that Ads wants to use this new feature, but we are encoding "cloud" into the package names?

Reply:
Ads may keep using it in the backward-compatible way, i.e. omitting the `assembly_name` prefix such that the `name` itself will be used for package_dir (and they can specify anything there with or without cloud prefix).